yarn.lock

```diff
@@ -0,0 +1,6094 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5":
+  version "7.5.5"
+  resolved "https://artifactory.availity.com/artifactory/api/npm/npm/@babel/code-frame/-/code-frame-7.5.5.tgz#bc0782f6d69f7b7d49531219699b988f669a8f9d"
```

Comment:
Should not be pointing to our proxy actually.

Reply:
I need to run unproxy then install again.